optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/domain/MeetingSchedule.java

```diff
@@ -44,6 +45,8 @@
 
     @XStreamConverter(HardMediumSoftScoreXStreamConverter.class)
     private HardMediumSoftScore score;
+    @ProblemFactProperty
+    private MeetingParametrization parametrization;
```

Comment:
Usually the parametrization field is the first or second field. The score is usually the last field.

Motivation: fields are more-or-less in order of volatility: fields that change often are later than fields that don't change often.
Motivation 2: a reliable, consistent order of fields and method reduces the chance of deadlocks (someone grabs A-B, someone else grabs B-A) and generally makes it harder for bugs to hide themselves (because a missing method call is eaiser spotted if the fields are always accessed in the same order)